### PR TITLE
Fix external tests

### PR DESCRIPTION
**Pull request by PavelPunegov**
* Don't run tests targeted to API < 1.4
* Fix importing classes in packages replacements during external test processing

### CODE REVIEW

File: build-tools/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy [hunk 1]
```diff
@@ -542,6 +544,11 @@ fun runTest() {
             return false
         }
 
+        def apiVersion = findLinesWithPrefixesRemoved(text, '// !API_VERSION: ')
+        if (apiVersion.size() != 0 && !apiVersion.contains("1.4")) {
```
Comment (ilmat192) [inline build-tools/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy]: Shouldn't we update the same check for language version above?

Comment (olonho) [reply build-tools/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy]: Maybe rewrite that logic in Kotlin finally?

Comment (PavelPunegov) [reply build-tools/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy]: I'm going to rewrite this and there is something already done in this way in another branch. But this fix is just to fix tests failures now.

As for language_version, I don't think this is necessary as tests pass.

Comment (ilmat192) [reply build-tools/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy]: > As for language_version, I don't think this is necessary as tests pass.

But currently we exclude all tests with LANGUAGE_VERSION=1.4, don't we?

**Pull request merged**